fdbserver/storageserver.actor.cpp

```diff
@@ -4645,6 +4661,10 @@ ACTOR Future<Void> fetchKeys(StorageServer* data, AddingShard* shard) {
 		    .detail("StorageVersion", data->storageVersion());
 		validate(data);
 
+		// the minimal version in updates must be larger than fetchVersion
+		if (!shard->updates.empty())
+			ASSERT(shard->updates[0].version > fetchVersion);
```

Comment:
change to `ASSERT(shard->updates.empty() || shard->updates[0].version > fetchVersion);'